x/bank/internal/keeper/integration_test.go

```diff
@@ -0,0 +1,22 @@
+package keeper_test
+
+import (
+	abci "github.com/tendermint/tendermint/abci/types"
+
+	"github.com/cosmos/cosmos-sdk/simapp"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/x/auth"
+)
+
+func createTestApp(isCheckTx bool) (*simapp.SimApp, sdk.Context) {
+	app := simapp.Setup(isCheckTx)
+	ctx := app.BaseApp.NewContext(isCheckTx, abci.Header{})
+
+	blacklistedAddrs := make(map[string]bool)
+	blacklistedAddrs[sdk.AccAddress([]byte("moduleAcc")).String()] = true
```

Comment:
Actually my bad, but this is not a valid address. The name should be hashed previously

Reply:
yea, will remove this. I think it is better to use the already black listed module account such as the distribution account then construct a new one